war/src/main/js/sortable-drag-drop.js

```diff
@@ -23,6 +23,14 @@ function registerSortableDragDrop(e) {
       forceFallback: true, // Do not use html5 drag & drop behaviour because it does not work with autoscroll
       scroll: true,
       bubbleScroll: true,
+      onChoose: function (event) {
+        const draggableDiv = event.item;
+        const height = draggableDiv.clientHeight;
+        draggableDiv.style.height = `${height}px`;
```

Comment:
We claim to still support IE 11 per https://www.jenkins.io/doc/administration/requirements/web-browsers/. If this is https://caniuse.com/template-literals, then this won't work.

Reply:
Off topic but is there a date in mind for when IE11 support will be dropped?

Reply:
No idea. Something for the UX sig I guess?

Reply:
I've added to the agenda of the next meeting on Wednesday https://docs.google.com/document/d/1QttPwdimNP_120JukigKsRuBvMr34KZhVfsbgq1HFLM/edit#heading=h.5w6rztvl5s02

Reply:
and mailing list: https://groups.google.com/g/jenkinsci-dev/c/piANoeohdik/m/PewPuvLSAAAJ